app/src/pages/TokenPage/TokenPage.tsx

```diff
@@ -87,6 +91,9 @@ export const TokenPage = (): React.ReactElement => {
     const tokenSalesCount = tokenSales ? tokenSales.length : 0;
     notdClient.getTokenRecentSales(registryAddress, tokenId, tokenSalesCount).then((tokenTransfers: TokenTransfer[]): void => {
```

Comment:
you should be passing 10 here otherwise if the default changes the paging would break